devdoc/Workunits.md

```diff
@@ -535,7 +429,7 @@ This means that it is valid to execute them, but they haven't been
 executed yet. Typically, only a few items begin with the state
 WFStateReqd.
 
-There are various specialised types of workflow items - e.g. sequential,
+There are various specialised types of workflow items - e.g., sequential,
```

Comment:
Since we are using US English spelling , specialised should be specialized

Reply:
I think I have to run US spell check against it once again.